tests/test_vtp.rb

```diff
@@ -304,19 +283,19 @@ def test_vtp_version_default
                  'Error: vtp version not default')
   end
 
-  def test_vtp_version_auto_enable
+  def test_version_auto_enable
     vtp = Vtp.new(false)
-    refute(Vtp.enabled, 'VTP should not be enabled')
+    refute(Feature.vtp_enabled?, 'VTP should not be enabled')
     vtp.version = 1
-    assert(Vtp.enabled)
+    assert(Feature.vtp_enabled?)
     assert_equal(1, vtp.version)
   end
 
-  def test_vtp_feature_enable_disable
-    Vtp.new.enable
-    assert(Vtp.enabled, 'Error: vtp is not enabled')
+  def test_feature_enable_disable
```

Comment:
Instead of removing this testcase I added a test for `Feature.vtp_disable`.  I considered adding it to `test_feature.rb` but figured here was better since it's a special case.